content/how-to/cloud-backup/rackspace-cloud-backup-backup-actions/index.md

```diff
@@ -24,17 +24,17 @@ You can access the **Backup Actions** menu in the following ways:
     on the gear icon next to the backup name on the **Configured Backups**
     page.
 
-    {{<image src="ScreenShot2015-10-15at4.20.11PM.png" alt="" title="">}}
+    {{<image src="ScreenShot2022-08-09at4.20.11PM.png" alt="" title="">}}
 
 -   Click on the **Actions** menu at the top of the **Configuration Details**
     page for a configured backup.
 
-    {{<image src="ScreenShot2015-10-15at4.18.36PM.png" alt="" title="">}}
+    {{<image src="ScreenShot2022-08-09at4.18.36PM.png" alt="" title="">}}
```

Comment:
rename to something descriptive would help. Like `cloud-backup-actions` if needed.
or just the article title and a number at the end.

Reply:
I also noticed an IP address in the image. Keep in mind these articles are visible to anyone on the internet. So if this would be a security risk, then you can photoshop the number or manipulate the HTML in the browser inspector to change the IP Address and then take the screenshot.

Reply:
resolved